sync-gateway/Dockerfile

```diff
@@ -8,6 +8,7 @@ LABEL maintainer="build-team@couchbase.com"
 RUN set -x \
     && microdnf update \
     && microdnf install shadow-utils \
+    && microdnf install systemd \
```

Comment:
What is this required for? systemd doesn't generally work in containers; does it bring in some required scripts or something?

Reply:
Saw systemd error during docker build even though it didn't fail.  Since systemctl is not need to start the docker image, it is probably okay to go w/o it.  End users won't notice at all unless he/she tries to build the image from our Dockerfile, which is unlike.  We can probably go w/o this PR.